examples/fun-op-sem/imp/impScript.sml

```diff
@@ -104,6 +104,25 @@ val cval_ind = prove(
 
 Theorem cval_ind[allow_rebind] = cval_ind
 
+Theorem seq_none:
```

Comment:
How about first proving the more general rewrite:
```
    cval (Seq c1 c2) s t = NONE <=> 
        cval c1 s t = NONE \/
        ?s' t'. cval c1 s t = SOME (s',t') /\
                  cval c2 s' t' = NONE
```
Your implication is an easy consequence of this, and the above is generally useful as a rewrite.

Reply:
Should this be a simp as well?

Reply:
That's harder to say; it may cause a blowup in the goal because the `cval c1` term is duplicated.  My feeling is that it probably shouldn't.